OpenSCAP/scap-security-guide
# Fix Rule ensure logrotate activated

#### Description:

- Add test scenarios for Rule ensure_logrotate_activated
- Update OVAL definition to:
  - Allow other arguments to logrotate command
  - Ensure that `/etc/logrotate.conf` is the last config file
  - Check for both `logrotate.conf` AND `cron.daily`
- Add bash remediation
